modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/human.dm

```diff
@@ -210,106 +187,80 @@
 		else
 			return feet_count
 
+/// Gets the number of feet the human has.
 /mob/living/carbon/human/proc/get_num_feet()
 	return has_feet(REQUIRE_GENITAL_ANY)
 
-// Weird procs go here
+/// Returns true if the human has a accessible ears for the parameter. Accepts any of the `REQUIRE_GENITAL_` defines.
 /mob/living/carbon/human/proc/has_ears(required_state = REQUIRE_GENITAL_ANY)
-	var/obj/item/organ/peepee = getorganslot(ORGAN_SLOT_EARS)
-	if(peepee)
-		switch(required_state)
-			if(REQUIRE_GENITAL_ANY)
-				return TRUE
-			if(REQUIRE_GENITAL_EXPOSED)
-				if(get_item_by_slot(ITEM_SLOT_EARS))
-					return FALSE
-				else
-					return TRUE
-			if(REQUIRE_GENITAL_UNEXPOSED)
-				if(!get_item_by_slot(ITEM_SLOT_EARS))
-					return FALSE
-				else
-					return TRUE
-			else
-				return TRUE
-	return FALSE
+	var/obj/item/organ/genital = getorganslot(ORGAN_SLOT_EARS)
+	if(!genital)
+		return FALSE
+	switch(required_state)
+		if(REQUIRE_GENITAL_ANY)
+			return TRUE
+		if(REQUIRE_GENITAL_EXPOSED)
+			return !get_item_by_slot(ITEM_SLOT_EARS)
+		if(REQUIRE_GENITAL_UNEXPOSED)
+			return get_item_by_slot(ITEM_SLOT_EARS)
+		else
+			return TRUE
 
+/// Returns true if the human has accessible ear sockets for the parameter. Accepts any of the `REQUIRE_GENITAL_` defines.
 /mob/living/carbon/human/proc/has_earsockets(required_state = REQUIRE_GENITAL_ANY)
```

Comment:
This is literally `has_ears`, unless I'm missing something. Feels unnecessary.

Reply:
Also unused.